platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/LogoutState.java

```diff
@@ -97,4 +103,22 @@ public String getCurrentRequestId() {
   public void setCurrentRequestId(String currentRequestId) {
     this.currentRequestId = currentRequestId;
   }
+
+  public List<String> getSessionIndexes() {
+    return sessionIndexes;
+  }
+
+  @SuppressWarnings("unused")
+  public void setSessionIndexes(List<String> sessionIndexes) {
+    if (Objects.nonNull(sessionIndexes)) {
```

Comment:
❓ I've noticed `Objects.nonNull` being used instead of simply doing `sessionIndexes != null` for instance. Curious to know why?

Reply:
I started using those methods when I started using Java streams:
```
    .stream()
        .filter(Objects::nonNull)
        .filter(s -> Objects.nonNull(s.getBinding()))
```
And I have a philosophical preference for method dispatch over build-in operators. But you reminded me it is not idiomatic and looks strange. I replaced them with `!=`